src/Microsoft.Health.Fhir.Liquid.Converter/BaseProcessor.cs

```diff
@@ -49,6 +51,36 @@ protected virtual Context CreateContext(ITemplateProvider templateProvider, IDic
             return context;
         }
 
+        protected virtual void CreateTraceInfo(IDictionary<string, object> data, TraceInfo traceInfo)
```

Comment:
>virtual [](start = 18, length = 7)

change to abstract?

Reply:
I think virtual might be more suitable here. The default behavior is no action. HL7 v2 overrides this and C-CDA and JSON just use the default. If we change it to abstract, all processors will need to implement their own `CreateTraceInfo`.